src/Microsoft.Data.Analysis/IDataView.Extension.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Microsoft.Data.Analysis
```

Comment:
I wonder if this should be `Microsoft.ML`, where `IDataView` is. That way it will be there without the user needing to include a different namespace.

Thoughts?

Reply:
Im worried about that being a breaking change for any of the existing users.

Reply:
My bad. I realize now you are meaning only for the new extensions. I think keeping it with the current namespace makes sense because they can't use the dataframe with including this namespace anyways right?

Reply:
> they can't use the dataframe with including this namespace anyways right?

No, you totally can. For example:

```C#
using Microsoft.ML;

...

IDataView scoredData = ....;
var dataFrame = scoredData.ToDataFrame();

Console.WriteLine(dataFrame.Columns.Count);
```

Here I didn't need to include the `Microsoft.Data.Analysis` namespace.

Reply:
> I wonder if this should be Microsoft.ML, where IDataView is.

This is a good idea. I hadn't considered this at all, but the more I think about it, the more sense it makes.